include/components/player_component.hpp

```diff
@@ -0,0 +1,15 @@
+#ifndef FILE_PLAYER_COMPONENT_HPP
```

Comment:
This component should be in the Engine

Reply:
It should be in the game because in the future it will contain stuff like player name which is only for the game

Reply:
@Bram-Boris Do you agree with Jan, otherwise this conversation should be re-opened.